samples/server/petstore/kotlin-springboot/src/main/kotlin/org/openapitools/api/PetApi.kt

```diff
@@ -56,7 +56,7 @@ class PetApiController(@Autowired(required = true) val service: PetApiService) {
     @RequestMapping(
             value = ["/pet/{petId}"],
             method = [RequestMethod.DELETE])
-    fun deletePet(@ApiParam(value = "Pet id to delete", required=true) @PathVariable("petId") petId: Long,@ApiParam(value = "" ) @RequestHeader(value="api_key", required=false) apiKey: String?): ResponseEntity<Unit> {
+    fun deletePet(@ApiParam(value = "Pet id to delete", required=true, defaultValue="null") @PathVariable("petId") petId: Long,@ApiParam(value = "" , defaultValue="null") @RequestHeader(value="api_key", required=false) apiKey: String): ResponseEntity<Unit> {
```

Comment:
Looks like the default should be `null` (without double quotes) or should be omitted.

I don't think it's related to this change so we will need to fix it later.

Reply:
I saw lot of change in the gérérated code (last time it was générated was 3.3.9 on Sep 2018).
i alsa saw a change in the body parameter name 
`(@Valid` @RequestBody pet: Pet 
changed to 
`@Valid` @RequestBody body: Pet

Reply:
there is another PR for this problem  i think
#2487